google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -333,7 +335,7 @@ def _api_client_factory(storage, config, parallel):
     if config.get_cai_enabled():
         # TODO: When CAI supports resource exclusion, update the following
         #       method to handle resource exclusion during export time.
-        asset_count = cloudasset.load_cloudasset_data(storage.session, config)
+        asset_count = cloudasset.load_cloudasset_data(storage.session, config, tracer)
```

Comment:
No need to pass tracer explicitly here, since this method call is in the same context as the calling method. The tracer will be inferred from context and set to the `tracer` argument of the method automatically.

Reply:
The reason why I passed this tracer in here is because I need to pass in the tracer into the lower level `_export_assets()` method that is inside `load_cloudasset_data()`.  So, inside `load_cloudasset_data()`, how do I get a hold of the tracer?

Reply:
Since `load_cloudasset_data()` has the `tracing.trace()` decorator and the `tracer=None` arg in its definition, the current context tracer will be injected by the decorator when calling the function, and you can just access it within the function using the `tracer` arg.

Reply:
Hmmm, although not passing in the tracer works here, I still can not see the lower level `_export_assets()` in the tracing output (see below).  What would be the way to make that show up?  Note: I am passing the tracer into `export_assets()`, inside the `load_cloudasset_data()`.  @ocervell 
  
![image](https://user-images.githubusercontent.com/6977544/64969716-4dc5ac00-d859-11e9-9718-38ff7ce9f286.png)